docs/data/xrootd/install-client.md

```diff
@@ -1,3 +1,6 @@
+!!!bug "EL7 version compatibility"
+    There is an incompatibility with EL7 < 7.5 due to an issue with the xrootd-lcmaps package
```

Comment:
Looking over https://support.opensciencegrid.org/public/tickets/82cb437a76f16b0ba8b4b97dd1179d79af28fe71cec6b80104fb5069bade2e74, we can be a bit more specific:

```suggestion
    There is an incompatibility with EL7 < 7.5 due to an issue with the `globus-gsi-proxy-core` package
```

Could you change the other xrootd-lcmaps one?

Reply:
I added a new commit with the change